node-notes-cli/app.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+
+const jsonFile = require('./data.json');
+const currentID = jsonFile.nextId;
+
+function read() {
+  for (const entry in jsonFile.notes) {
+    console.log(JSON.stringify(`${entry}: ${jsonFile.notes[entry]}`));
```

Comment:
The `JSON.stringify` is counter-productive, since it adds quotes to the string, and the target output doesn't have quotes:

---

![image](https://user-images.githubusercontent.com/16143771/214218863-151543a4-a404-4135-994f-594f933ca55e.png)